node/node.go

```diff
@@ -619,7 +626,14 @@ func validateShardBoundMessage(consensus *consensus.Consensus, peer libp2p_peer.
 		if maybeVC.ViewId+5 < consensus.GetViewChangingID() {
 			return nil, nil, true, errors.WithStack(errViewIDTooOld)
 		}
-	} else {
+	case maybeSP != nil:
+		if maybeSP.ShardId != consensus.ShardID {
+			nodeConsensusMessageCounterVec.With(prometheus.Labels{"type": "invalid_shard"}).Inc()
+			return nil, nil, true, errors.WithStack(errWrongShardID)
+		}
+		senderKey = maybeSP.SenderPubkey
+		consensus.SetLastKnownSignPower(maybeSP.Commit)
```

Comment:
we need to capture all 3 VP :
Prepare:      getOrZero(prep),
Commit:       getOrZero(comm),
Change:       getOrZero(view),

during an outage, most likely we won't even hit the commit phase, and during that time only view change VP will be seen.